### DISCUSSION TITLE
Oqtane Forums?

### DISCUSSION THREAD

**Discussion by markdav-is** [DNNCommunity/Dnn.CommunityForums#1036] [Ideas]
I have a few Oqtane projects that would greatly benefit from CommunityForums style forums.  I was directed here from the Oqtane Github Discussions.  Is there anyone on this project that's interested in porting Community Forums to Oqtane?  I'll create a fork and start poking around to see how big a lift this might be. https://github.com/markdav-is/Dnn.CommunityForums.

**Comment by rchibnik** [comment]
I have no idea how to port this, but I'm interested in migrating to Oqtane. I've done a bit of Oqtane development but not any DNN. At me if you have a plan.

**Comment by zyhfish** [comment]
I will try to check whether I can work on this, IMO we can reuse the database structure and styles, so that we can focus on the backend code translation.

**Comment by rchibnik** [comment]
That makes sense. I have the project source but don’t have a dnn environment To work with
